old-feed.user.js

```diff
@@ -14,15 +14,19 @@
 
 (function() {
     'use strict';
+
     const feedContainer = document.querySelector("#dashboard feed-container");
     // Apparently if this isn't true, then a SSO popup is being shown, so don't do anything.
     if (!feedContainer) return;
 
     const columnContainer = document.querySelector(".feed-content");
     columnContainer.classList.remove("flex-justify-center");
-    columnContainer.style.maxWidth="100vw";
+    columnContainer.style.maxWidth="unset";
     const feedColumn = columnContainer.querySelector(".feed-main");
-    feedColumn.style.maxWidth="100vw";
+    feedColumn.style.maxWidth="100%";
+    const sidebar = document.querySelector('.feed-right-sidebar');
+    sidebar.style.maxWidth = "unset";
+    sidebar.style.width = "900px";
```

Comment:
Why this change? With a wide monitor, the left and right sidebars no longer match in width... looks weird to me.

Reply:
You can skip this change if it looks bad on wide monitors. I tested only on one screen.